app/assets/javascripts/student_profile_v2/profile_details.js

```diff
@@ -3,14 +3,46 @@
   var dom = window.shared.ReactHelpers.dom;
   var createEl = window.shared.ReactHelpers.createEl;
   var merge = window.shared.ReactHelpers.merge;
-
+  var FeedHelpers = window.shared.FeedHelpers;
   var QuadConverter = window.shared.QuadConverter;
 
+  var Colors = {
+    math: '#e8e9fc',
+    reading: '#ffe7d6',
+    procedural: '#e8fce8'
+  };
```

Comment:
Cool, good to know where styles go. I'll do that as a separate pass, since I'm real tired.